core/config/project_settings.cpp

```diff
@@ -1235,6 +1235,20 @@ ProjectSettings::ProjectSettings() {
 
 	GLOBAL_DEF("compression/formats/gzip/compression_level", Compression::gzip_level);
 	custom_prop_info["compression/formats/gzip/compression_level"] = PropertyInfo(Variant::INT, "compression/formats/gzip/compression_level", PROPERTY_HINT_RANGE, "-1,9,1");
+
+	GLOBAL_DEF("debug/settings/crash_handler/message",
+			String("Please include this when reporting the bug on https://github.com/godotengine/godot/issues"));
+	GLOBAL_DEF_RST("rendering/occlusion_culling/bvh_build_quality", 2);
+	GLOBAL_DEF("memory/limits/multithreaded_server/rid_pool_prealloc", 60);
+	GLOBAL_DEF("internationalization/rendering/force_right_to_left_layout_direction", false);
+
+	GLOBAL_DEF("gui/timers/incremental_search_max_interval_msec", 2000);
+	ProjectSettings::get_singleton()->set_custom_property_info("gui/timers/incremental_search_max_interval_msec", PropertyInfo(Variant::INT, "gui/timers/incremental_search_max_interval_msec", PROPERTY_HINT_RANGE, "0,10000,1,or_greater")); // No negative numbers.
+
+	GLOBAL_DEF("rendering/vulkan/staging_buffer/block_size_kb", 256);
+	GLOBAL_DEF("rendering/vulkan/staging_buffer/max_size_mb", 128);
+	GLOBAL_DEF("rendering/vulkan/staging_buffer/texture_upload_region_size_px", 64);
+	GLOBAL_DEF("rendering/vulkan/descriptor_pools/max_descriptors_per_pool", 64);
```

Comment:
Well they should stay where they were in the first place IMO, next to the other `rendering/vulkan/` settings highlighted here:
![image](https://user-images.githubusercontent.com/4701338/176209408-c2a414fd-e74e-43b0-a401-d48bfeecd192.png)

And next to all other `rendering/` settings.

Reply:
Well, according to the comment, they can't be defined here, because there is other code that runs earlier and depends on these settings being defined. And I can't move them to the other place, because they won't be picked by doctool.

So they need to be moved to a new place. I could maybe move other settings too.

Reply:
No the comment means that they used to be defined only in `RenderingDeviceVulkan`, but that's only part of the doctool code path if you run with Vulkan, and not with `--headless`. So they were duplicated here in `RenderingServer` (which always gets constructed regardless of options).

So you just need to remove the ones in `RenderingDeviceVulkan`.